libs/@hashintel/design-system/src/type-card.tsx

```diff
@@ -0,0 +1,142 @@
+import { faArrowsRotate, faAsterisk } from "@fortawesome/free-solid-svg-icons";
+import {
+  Box,
+  Collapse,
+  SvgIcon,
+  SvgIconProps,
+  Tooltip,
+  Typography,
+} from "@mui/material";
+import { ElementType, FunctionComponent, useState } from "react";
+
+import { FontAwesomeIcon } from "./fontawesome-icon";
+import { IconButton } from "./icon-button";
+import { WhiteCard } from "./white-card";
+
+interface TypeCardProps {
+  onDelete?: () => void;
```

Comment:
This is mostly just moved from the type editor, but this `onDelete` is new (it's the 'x' that appears if you hover over an inherited type)